x-pack/test/api_integration/apis/security/change_password.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Cookie, cookie } from 'request';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getService }: FtrProviderContext) {
+  const supertest = getService('supertestWithoutAuth');
+
+  const mockUserName = 'test-user';
+  const mockUserPassword = 'test-password';
+
+  describe('Change password', () => {
+    let sessionCookie: Cookie;
+    beforeEach(async () => {
+      // Create mock user to change password for.
+      await getService('supertest')
```

Comment:
optional: we added a security service which potentially makes this a bit easier https://github.com/elastic/kibana/blob/d66b3c74eb18d72208d0652a0b5f72cd9701191b/x-pack/test/api_integration/apis/apm/feature_controls.ts#L190-L194

Reply:
wow, didn't know about that, thanks!